### DISCUSSION TITLE
Domains not found in the domain tab

### DISCUSSION THREAD

**Discussion by y1xing** [vercel/community#6539] [Help]
### Summary

Recently vercel seemed to have created a new team for me and transferred all the projects from my initial dashboard to the dashboard with the teams. I clicked on the domain tab and all my domain was lost. I am unable to get back to my initial dashboard. How can I get back my domains?

### Steps to Reproduce

Go to domain tab

**Comment by swarnava** [comment]
Hi,

Can you check here: https://vercel.com/account/domains?
